MultiModalCLF/MoodBoardMultiModalClassification.ipynb

```diff
@@ -0,0 +1,3617 @@
+            "/usr/local/lib/python3.10/dist-packages/torchvision/models/_utils.py:223: UserWarning: Arguments other than a weight enum or `None` for 'weights' are deprecated since 0.13 and may be removed in the future. The current behavior is equivalent to passing `weights=ResNet18_Weights.IMAGENET1K_V1`. You can also use `weights=ResNet18_Weights.DEFAULT` to get the most up-to-date weights.\n",
+            "  warnings.warn(msg)\n",
+            "Downloading: \"https://download.pytorch.org/models/resnet18-f37072fd.pth\" to /root/.cache/torch/hub/checkpoints/resnet18-f37072fd.pth\n",
+            "100%|██████████| 44.7M/44.7M [00:00<00:00, 154MB/s]\n"
+          ]
+        },
+        {
+          "output_type": "display_data",
+          "data": {
+            "text/plain": [
+              "config.json:   0%|          | 0.00/1.18k [00:00<?, ?B/s]"
+            ],
+            "application/vnd.jupyter.widget-view+json": {
+              "version_major": 2,
+              "version_minor": 0,
+              "model_id": "f130554d609f405b9158249ca39f273b"
+            }
+          },
+          "metadata": {}
+        },
+        {
+          "output_type": "display_data",
+          "data": {
+            "text/plain": [
+              "pytorch_model.bin:   0%|          | 0.00/438M [00:00<?, ?B/s]"
+            ],
+            "application/vnd.jupyter.widget-view+json": {
+              "version_major": 2,
+              "version_minor": 0,
+              "model_id": "20a6612d75d7488580b0c04551968c21"
+            }
+          },
+          "metadata": {}
+        },
+        {
+          "output_type": "stream",
+          "name": "stderr",
+          "text": [
+            "/usr/local/lib/python3.10/dist-packages/torch/_utils.py:831: UserWarning: TypedStorage is deprecated. It will be removed in the future and UntypedStorage will be the only storage class. This should only matter to you if you are using storages directly.  To access UntypedStorage directly, use tensor.untyped_storage() instead of tensor.storage()\n",
+            "  return self.fget.__get__(instance, owner)()\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "from sklearn.metrics import accuracy_score\n",
+        "import numpy as np\n",
+        "\n",
+        "# Calculates what percentage of labels were predicted correctly\n",
+        "def compute_metrics(p):\n",
+        "    preds = np.argmax(p.predictions, axis=1)\n",
+        "    accuracy = accuracy_score(p.label_ids, preds)\n",
+        "    print(\"Accuracy:\", accuracy)\n",
+        "    return {\"accuracy\": accuracy}"
+      ],
+      "metadata": {
+        "id": "hqob-r_KB4og"
+      },
+      "execution_count": 13,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "training_args = TrainingArguments(\n",
+        "    num_train_epochs=10,\n",
+        "    learning_rate=5e-6,\n",
+        "    weight_decay=0.01,\n",
```

Comment:
How did you pick these parameters? I'm interested to see how tuning these values will affect the loss and accuracy